language_formatters_pre_commit_hooks/pretty_format_kotlin.py

```diff
@@ -58,13 +58,6 @@ def pretty_format_kotlin(argv: typing.Optional[typing.List[str]] = None) -> int:
     parser.add_argument("filenames", nargs="*", help="Filenames to fix")
     args = parser.parse_args(argv)
 
-    # KTLint does not yet support Java 16+, before that version.
-    # Let's make sure that we report a nice error message instead of a complex
-    # Java Stacktrace
-    # the tool can only be executed on Java up to version 15.
-    # Context: https://github.com/JLLeitschuh/ktlint-gradle/issues/461
-    assert_max_jdk_version(Version("16.0"), inclusive=False)  # pragma: no cover
```

Comment:
Please remove the `@pytest.mark.skipif` annotations in the tests

https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/f5b509c82c6b8d34a7c971e28edf102b3c5ec987/tests/pretty_format_kotlin_test.py#L48
https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/f5b509c82c6b8d34a7c971e28edf102b3c5ec987/tests/pretty_format_kotlin_test.py#L53
https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/f5b509c82c6b8d34a7c971e28edf102b3c5ec987/tests/pretty_format_kotlin_test.py#L58

Reply:
Sorry, I missed the tests in the first place. Done.